build/images/webhook/Dockerfile

```diff
@@ -2,11 +2,13 @@ FROM golang:1.16 as audit-build
 WORKDIR /antrea
 COPY . /antrea
 RUN CGO_ENABLED=0 go build -o audit-webhook ./cmd/webhook
+RUN CGO_ENABLED=0 go build -o auditctl ./cmd/cli
 
-FROM scratch
+FROM ubuntu:20.04
```

Comment:
I understand this change, but it does bother me a little that we can no longer use a distroless image. Maybe add a TODO that we can revert back to distroless when we support running `auditctl` from outside of the cluster.